File: mario-more/mario.c
// Pyramids generator that accepts input between 1 and 8

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get user input 1-8
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Generate pyramids
    for (int i = 0; i < height; i++)
    {
        // spaces for left alignment
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }

        // hashes for left pyramid
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        // print static space
        printf("  ");

        // print hashes for right pyramid
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
    }

}

File: sentimental-mario-more/mario.py
# Print half-pyramid from Problem Set 1

from cs50 import get_int


def main():
    """Get height from user"""
    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            break

    """Print pyramids"""
    for i in range(height):
        # Print spaces for left pyramid
        print(" " * (height - i - 1), end="")

        # Print hashes for left pyramid
        print("#" * (i + 1), end="")

        # Print 2 spaces separation
        print("  ", end="")

        # Print hashes for right side
        print("#" * (i + 1))


main()
